### DISCUSSION TITLE
Same Object - different icon / different color

### DISCUSSION THREAD

**Discussion by simon-paycaptain** [deejay-hub/timeline-lwc#182] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/89647059/163578476-908731c2-e156-4f64-87da-1c025ea2a303.png)

See image - we would like to be able to show items on the timeline for the same object (with different criteria) - but different icons and different colors.  I tried to create a separate Timeline Configuration record - but it does not work.    Is this a known limitation - any plans to allow this use case?

**Comment by deejay-hub** [comment, marked as answer]
Hi @simon-paycaptain, Yes this is a known limit. We have a section in the FAQs that mentions only one child object is allowed at the moment. 
https://quip.com/6yvoAcBukqZB
See section - *Can I plot the same Object multiple times to use different icons and fields? Perhaps based on record type or status as an example.*

We also have an issue raised asking to add the functionality (https://github.com/deejay-hub/timeline-lwc/issues/27). So that's the issue to track for future releases if we address the request. It's a good candidate to be added but since it's a community component that's free there is no designated timeline.
